assets/sample_docs/syntax.json

```diff
@@ -0,0 +1,9 @@
+{
+  // some comment
+  "internal_command": "#ff0000|#ffffff|b",
+  // some other comment
```

Comment:
nit: json doesn't support comments (technically)

Reply:
it does with our json crate